diff --git a/gen/lib/Okta/Model.hs b/gen/lib/Okta/Model.hs
index 2b3977e..eed4fd4 100644
--- a/gen/lib/Okta/Model.hs
+++ b/gen/lib/Okta/Model.hs
@@ -12968,6 +12968,7 @@ data User = User
   , userActivated :: !(Maybe DateTime) -- ^ /ReadOnly/ "activated"
   , userCreated :: !(Maybe DateTime) -- ^ /ReadOnly/ "created"
   , userCredentials :: !(Maybe UserCredentials) -- ^ "credentials"
+  , userGroupIds :: !(Maybe [Text]) -- ^ "groupIds"
   , userId :: !(Maybe Text) -- ^ /ReadOnly/ "id"
   , userLastLogin :: !(Maybe DateTime) -- ^ /ReadOnly/ "lastLogin"
   , userLastUpdated :: !(Maybe DateTime) -- ^ /ReadOnly/ "lastUpdated"
@@ -12988,6 +12989,7 @@ instance A.FromJSON User where
       <*> (o .:? "activated")
       <*> (o .:? "created")
       <*> (o .:? "credentials")
+      <*> (o .:? "groupIds")
       <*> (o .:? "id")
       <*> (o .:? "lastLogin")
       <*> (o .:? "lastUpdated")
@@ -13007,6 +13009,7 @@ instance A.ToJSON User where
       , "activated" .= userActivated
       , "created" .= userCreated
       , "credentials" .= userCredentials
+      , "groupIds" .= userGroupIds
       , "id" .= userId
       , "lastLogin" .= userLastLogin
       , "lastUpdated" .= userLastUpdated
@@ -13029,6 +13032,7 @@ mkUser =
   , userActivated = Nothing
   , userCreated = Nothing
   , userCredentials = Nothing
+  , userGroupIds = Nothing
   , userId = Nothing
   , userLastLogin = Nothing
   , userLastUpdated = Nothing
diff --git a/gen/lib/Okta/ModelLens.hs b/gen/lib/Okta/ModelLens.hs
index 19dfc44..284946b 100644
--- a/gen/lib/Okta/ModelLens.hs
+++ b/gen/lib/Okta/ModelLens.hs
@@ -7633,6 +7633,11 @@ userPasswordChangedL :: Lens_' User (Maybe DateTime)
 userPasswordChangedL f User{..} = (\userPasswordChanged -> User { userPasswordChanged, ..} ) <$> f userPasswordChanged
 {-# INLINE userPasswordChangedL #-}
 
+-- | 'userGroupIds' Lens
+userGroupIdsL :: Lens_' User (Maybe [Text])
+userGroupIdsL f User{..} = (\userGroupIds -> User { userGroupIds, ..} ) <$> f userGroupIds
+{-# INLINE userGroupIdsL #-}
+
 -- | 'userProfile' Lens
 userProfileL :: Lens_' User (Maybe UserProfile)
 userProfileL f User{..} = (\userProfile -> User { userProfile, ..} ) <$> f userProfile
