diff --git gen/lib/Okta/Model.hs gen/lib/Okta/Model.hs
index ec8fd70..0c7d2db 100644
--- gen/lib/Okta/Model.hs
+++ gen/lib/Okta/Model.hs
@@ -4131,6 +4131,7 @@ data UserP p = User
   , userActivated :: !(Maybe DateTime) -- ^ "activated"
   , userCreated :: !(Maybe DateTime) -- ^ "created"
   , userCredentials :: !(Maybe UserCredentials) -- ^ "credentials"
+  , userGroupIds :: !(Maybe [Text]) -- ^ "groupIds"
   , userId :: !(Maybe Text) -- ^ "id"
   , userLastLogin :: !(Maybe DateTime) -- ^ "lastLogin"
   , userLastUpdated :: !(Maybe DateTime) -- ^ "lastUpdated"
@@ -4150,6 +4151,7 @@ instance A.FromJSON p => A.FromJSON (UserP p) where
       <*> (o .:? "activated")
       <*> (o .:? "created")
       <*> (o .:? "credentials")
+      <*> (o .:? "groupIds")
       <*> (o .:? "id")
       <*> (o .:? "lastLogin")
       <*> (o .:? "lastUpdated")
@@ -4168,6 +4170,7 @@ instance A.ToJSON p => A.ToJSON (UserP p) where
       , "activated" .= userActivated
       , "created" .= userCreated
       , "credentials" .= userCredentials
+      , "groupIds" .= userGroupIds
       , "id" .= userId
       , "lastLogin" .= userLastLogin
       , "lastUpdated" .= userLastUpdated
@@ -4189,6 +4192,7 @@ mkUser =
   , userActivated = Nothing
   , userCreated = Nothing
   , userCredentials = Nothing
+  , userGroupIds = Nothing
   , userId = Nothing
   , userLastLogin = Nothing
   , userLastUpdated = Nothing
diff --git gen/lib/Okta/ModelLens.hs gen/lib/Okta/ModelLens.hs
index 1420cc0..1caa6bf 100644
--- gen/lib/Okta/ModelLens.hs
+++ gen/lib/Okta/ModelLens.hs
@@ -2428,6 +2428,11 @@ userPasswordChangedL :: Lens_' User (Maybe DateTime)
 userPasswordChangedL f User{..} = (\userPasswordChanged -> User { userPasswordChanged, ..} ) <$> f userPasswordChanged
 {-# INLINE userPasswordChangedL #-}
 
+-- | 'userGroupIds' Lens
+userGroupIdsL :: Lens_' User (Maybe [Text])
+userGroupIdsL f User{..} = (\userGroupIds -> User { userGroupIds, ..} ) <$> f userGroupIds
+{-# INLINE userGroupIdsL #-}
+
 -- | 'userProfile' Lens
 userProfileL :: Lens_' (UserP p) (Maybe p)
 userProfileL f User{..} = (\userProfile -> User { userProfile, ..} ) <$> f userProfile
